addons/skin.estuary/xml/Includes_PVR.xml

```diff
@@ -197,8 +197,19 @@
 				<width>200</width>
 				<height>200</height>
 				<aspectratio align="center" aligny="top">keep</aspectratio>
-				<texture>$INFO[Listitem.Icon]</texture>
+				<texture fallback="DefaultTVShows.png">$INFO[Listitem.Icon]</texture>
 				<fadetime>200</fadetime>
+				<visible>String.IsEmpty(Listitem.EpgEventIcon)</visible>
+			</control>
+			<control type="image">
+				<top>135</top>
+				<left>630</left>
+				<width>200</width>
+				<height>200</height>
+				<aspectratio align="center" aligny="top">keep</aspectratio>
+				<texture fallback="DefaultTVShows.png">$INFO[Listitem.EpgEventIcon]</texture>
+				<fadetime>200</fadetime>
+				<visible>!String.IsEmpty(Listitem.EpgEventIcon)</visible>
```

Comment:
i thought (perhaps i'm wrong) that ListItem.Icon will already display the EPG Event Icon if available?

in case i'm wrong and there is a usecase for this change, please use a $VAR[] instead of duplicating the image control.

Reply:
@ronie you are wrong. If it's a listitem for a channel (like in this case) there is no fallback to or preference of the event thumbnail. Only if it is a listitem for an epg event, there is a fallback to the channel logo if no epg thumb is available.

Reply:
(however invented this logic....)